lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -234,4 +237,44 @@ describe('SEO: Font size audit', () => {
     expect(auditResult.score).toBe(1);
     expect(auditResult.notApplicable).toBe(true);
   });
+
+  it('attributes inline and attributes styles to node', async () => {
+    const style1 = {
+      type: 'Inline',
+    };
+    const style2 = {
+      type: 'Attributes',
+    };
+    const parentNode = {
+      attributes: ['id', 'my-parent'],
+    };
+    const artifacts = {
+      URL,
+      MetaElements: makeMetaElements(validViewport),
+      FontSize: {
+        analyzedFailingNodesData: [
+          {textLength: 12, fontSize: 10, cssRule: style1,
+            node: {nodeId: 1, parentNode, localName: 'p', attributes: ['class', 'my-p']}},
+          {textLength: 11, fontSize: 10, cssRule: style2,
+            node: {nodeId: 2, parentNode, localName: 'font', attributes: ['size', '10px']}},
+        ],
+      },
+      TestedAsMobileDevice: true,
+    };
+
+    const auditResult = await FontSizeAudit.audit(artifacts, getFakeContext());
+    assert.equal(auditResult.details.items.length, 2);
+    assert.equal(auditResult.details.items[0].source, URL.finalUrl);
```

Comment:
`expect(auditResult.details.items).toMatchObject`

for the new stuff? :D

Reply:
done.

Also, I grabbed the `describe` fn I made in #9354 and moved it here. 1) will make the other review smaller 2) doing the same stuff here.